pkg/storage/mysql/mysql.go

```diff
@@ -24,11 +24,16 @@ import (
 
 	"github.com/openfga/openfga/pkg/logger"
 	"github.com/openfga/openfga/pkg/storage"
+	"github.com/openfga/openfga/pkg/storage/migrate"
 	"github.com/openfga/openfga/pkg/storage/sqlcommon"
 	tupleUtils "github.com/openfga/openfga/pkg/tuple"
 )
 
-var tracer = otel.Tracer("openfga/pkg/storage/mysql")
+var (
+	Migrations = migrate.NewRegistry("mysql")
```

Comment:
This belongs in `pkg/storage/mysql/migrations/migrations.go` not here up one level.